src/interaction/interactive_api.jl

```diff
@@ -168,9 +168,107 @@ function select_rectangle(scene; kwargs...)
                 end
             end
             # always hide if not the right key is pressed
-            rect_vis[:visible] = false # make the plotted rectangle invisible
+            plotted_rect[:visible] = false # make the plotted rectangle invisible
         end
         return rect_ret
     end
     return rect_ret
 end
+
+"""
+    select_line(scene; kwargs...) -> line
+Interactively select a line (typically an arrow) on a 2D `scene` by clicking the left mouse button,
+dragging and then un-clicking. Return an **observable** whose value corresponds
+to the selected line on the scene. In addition the function
+_plots_ the line on the scene as the user clicks and moves the mouse
+around. When the button is not clicked any more, the plotted line disappears.
+
+The value of the returned line is updated **only** when the user un-clicks.
+
+The `kwargs...` are propagated into `lines!` which plots the selected line.
+"""
+function select_line(scene; kwargs...)
+    key = Mouse.left
+    waspressed = Node(false)
```

Comment:
If you never lift on this, it could also be a `Ref`, although I don't know exactly how much overhead that shaves off.
Also, it'd be nice to use `Observable` instead of `Node`, so we can start to move away from that.

Reply:
I disagree: The healthy way to do it is to load up Atom and do "Find all Replace all" on `Node`. Doing it one by one seems insane to me.

Reply:
Fair enough, there aren't any API incompatibilities (since it's the same API) so that should be a 5-minute job.

Incidentally, what are your thoughts on `Obs` as a shorthand for `Observable`?

Reply:
I think it is great.